src/components/OurServicesSection/OurServices.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./ourservices.css";
+import OurServicesFeatures from "./OurServicesFeatures";
+
+export default function OurServices() {
+  return (
+    <div className="ourServicesContainer">
+      <div className="ourServicesWrapper">
+        <div className="servicesHeader">
+          <h2 className="servicesHeaderText">WHY CHOOSE US</h2>
+        </div>
+        <div className="serviceOffers">
+          {OurServicesFeatures.map((services) => {
+            return (
+              <div className="serviceOffer">
```

Comment:
I think you might like to check your console for error message also because your map item are not having a unique key